themes/default/layouts/product/esc.html

```diff
@@ -204,36 +204,36 @@ <h2>Frequently asked questions</h2>
         </div>
         <ul class="list-none p-0 mx-5 xl:ml-12" data-faq-type="faq-pricing">
             <li class="accordion-item text-2xl py-3">
-                {{ partial "accordian-header" (dict "text" "What is Pulumi Environments, Secrets and Configurations" "large_header" true) }}
+                {{ partial "accordian-header" (dict "text" "What is Pulumi Environments, Secrets and Configuration" "large_header" true) }}
                 <div class="accordion-item-body-no-animation text-base">
                     <p>
-                        Pulumi ESC (Environments, Secrets and Configurations), is an open source project and managed service as part of Pulumi Cloud that enables teams to manage hierarchical collections of configuration and secrets, and then consume those configuration and secrets from a variety of different infrastructure and application services.
+                        Pulumi ESC (Environments, Secrets and Configuration), is an [open source project](https://github.com/pulumi/esc) and managed service as part of Pulumi Cloud that enables teams to manage hierarchical collections of configuration and secrets, and then consume those configuration and secrets from a variety of different infrastructure and application services.
```

Comment:
Never "those configuration":

```suggestion
                        Pulumi ESC (Environments, Secrets and Configuration), is an [open source project](https://github.com/pulumi/esc) and managed service of Pulumi Cloud that enables teams to manage hierarchical collections of configuration and secrets and consume them from a variety of different infrastructure and application services.
```